Week3/strings-arrays.js

```diff
@@ -0,0 +1,37 @@
+//1.Strings
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+myString=myString.replace(/,/g, " "); // that's totally google answer i couldn't find what the g stands for
+console.log(myString);
+
+
+
+//2 Arrays
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+// add Mauro's favorite animal 'turtle' 
+favoriteAnimals[3]="turtle";
```

Comment:
Ah yes, technically this will work. but the correct way to do this is like this:

```
favoriteAnimals.push('turtle');
```

See [here](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Array/push) for more information!